test/integration/targets/elb_target/playbooks/roles/elb_target/tasks/main.yml

```diff
@@ -213,6 +226,20 @@
           - result.changed
           - not result.target_health_descriptions
 
+    - name: remove an udp unused target
+      elb_target:
+        target_group_name: "{{ tg_tcpudp_name }}"
+        target_id: "{{ ec2.instance_ids[0] }}"
+        state: absent
+        deregister_unused: true
+        <<: *aws_connection_info
+      register: result
+
+    - name: target group was deleted
+      assert:
+        that:
+          - result.changed
```

Comment:
This assertion is failing because there are no targets to be removed from `tg_tcpudp_name`.  Are you trying to delete the target group itself?  That would need to be done with `elb_target_group`.

Reply:
hmmm, fixed that! I think its good now